kubenow/gluster-s3/values.yaml

```diff
@@ -0,0 +1,19 @@
+# Default values for a Gluster-S3 deployment.
+# this is a yaml-formatted file.
+# declare variables to be passed into your templates.
+releaseName: s3-interface
+secretName: s3-secret
+
+account: account
+user: user
+password: 8VBdiwxL2s
+endpoint: "gluster-s3-service.default.svc.cluster.local:8080"
```

Comment:
it can be hardcoded, `gluster-s3-service.default` should be enough.

Reply:
@andersla do we deploy GlusterFS on the default namespace right?

Reply:
no we don't deploy on the default namespace. it's on heketi-gluster. moreover, in my opinion it's better to keep it like this since it adds much more flexibility in case of future architectural changes